tests/playwright/page-object-model/trust/contacts-page.ts

```diff
@@ -25,18 +40,26 @@ class ContactsPageAssertions extends BaseTrustPageAssertions {
   }
 
   async toSeeCorrectDfeContacts (): Promise<void> {
-    await expect(this.contactsPage.dfeContactsCard).toContainText('Keyshawn Hermiston')
-    await expect(this.contactsPage.dfeContactsCard).toContainText('Keyshawn.Hermiston@education.gov.uk')
-    await expect(this.contactsPage.dfeContactsCard).toContainText('Ayana Lueilwitz')
-    await expect(this.contactsPage.dfeContactsCard).toContainText('Ayana.Lueilwitz@education.gov.uk')
+    await expect(this.contactsPage.dfeContactsCard).toContainText(`Trust relationship manager ${this.contactsPage.currentTrust.trustRelationshipManager.fullName} ${this.contactsPage.currentTrust.trustRelationshipManager.email}`)
+    await expect(this.contactsPage.dfeContactsCard).toContainText(`SFSO (Schools financial support and oversight) lead ${this.contactsPage.currentTrust.sfsoLead.fullName} ${this.contactsPage.currentTrust.sfsoLead.email}`)
   }
 
   async toSeeCorrectTrustContacts (): Promise<void> {
-    await expect(this.contactsPage.trustContactsCard).toContainText('Tyler Welch')
-    await expect(this.contactsPage.trustContactsCard).toContainText('Tyler.Welch@abbeylaneacademiestrust.co.uk')
-    await expect(this.contactsPage.trustContactsCard).toContainText('Courtney Pacocha')
-    await expect(this.contactsPage.trustContactsCard).toContainText('Courtney.Pacocha@abbeylaneacademiestrust.co.uk')
-    await expect(this.contactsPage.trustContactsCard).toContainText('Lowell Hoppe')
-    await expect(this.contactsPage.trustContactsCard).toContainText('Lowell.Hoppe@abbeylaneacademiestrust.co.uk')
+    const requiredTrustContacts = ['Accounting Officer', 'Chair of Trustees', 'Chief Financial Officer']
+    const trustContacts = this.contactsPage.currentTrust.governors.filter(x => requiredTrustContacts.some(n => n === x.role))
+    trustContacts.sort((a, b) => a.role.localeCompare(b.role))
+    await expect(this.contactsPage.trustContactsCard).toContainText(`Accounting officer ${trustContacts[0].fullName} ${trustContacts[0].email}`)
+    await expect(this.contactsPage.trustContactsCard).toContainText(`Chair of trustees ${trustContacts[1].fullName} ${trustContacts[1].email}`)
+    await expect(this.contactsPage.trustContactsCard).toContainText(`Chief financial officer ${trustContacts[2].fullName} ${trustContacts[2].email}`)
+  }
+
+  async toSeeCorrectDfeContactsMissingInformationMessage (): Promise<void> {
+    await expect(this.contactsPage.dfeContactsCard).toContainText('SFSO (Schools financial support and oversight) lead No contact information available')
+  }
+
+  async toSeeCorrectTrustContactMissingEmailMessage (): Promise<void> {
+    const requiredTrustContacts = ['Chief Financial Officer']
+    const trustContacts = this.contactsPage.currentTrust.governors.filter(x => requiredTrustContacts.some(n => n === x.role))
```

Comment:
As there's only one contact with this role, it would be better to use `find` and get rid of the `requiredTrustContacts` array